docs/guides/no-suspense.md

```diff
@@ -0,0 +1,87 @@
+# No Suspense
+
+> The content of this page would be easier if you have taken a look on the _introduction/async_ part of the docs.
+
+As you may know, sometimes we need to wrap our components in suspense to use asynchronous atoms. Two kinds of atoms need Suspense.
```

Comment:
Maybe remove the "As you may know", It doesn't provide value to the explanation.
```suggestion
Sometimes we need to wrap our components in suspense to use asynchronous atoms. Two kinds of atoms need Suspense.
```